examples/vae/train.py

```diff
@@ -12,45 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 from absl import app
 from absl import flags
 
+from ml_collections.config_flags import config_flags
+
 import numpy as np
 import jax.numpy as jnp
 
 import jax
 from jax import random
 
-from flax import nn
-from flax import optim
-
 import tensorflow as tf
 import tensorflow_datasets as tfds
 
+from flax import nn
```

Comment:
I would recommend reverting this change as it is not related to the problem being solved. 

There are such linter occurrences in almost all the examples, but it would be nice to have a wider fix or follow up PR for the same.